### ISSUE TITLE
Preprocess Data

### ISSUE THREAD

**Issue by Jalen1** [Jalen1/Retinal-Disease-Classification#2]
- Load in data from kaggle dataset
- Split data into train and test sets with their corresponding labels
- Transform png images into Tensors that can be passed into a PyTorch neural network
